designs/developer/topology-45/index.md

```diff
@@ -0,0 +1,298 @@
+---
+parent: Developer
+version: 4.5
```

Comment:
@Veethika i'm unsure what's different in 4.5 based on this index.md, but I could definitely have missed it!

Reply:
Removed the topology 4.5 folder. It would redirect to 4.4 instead. Also figured the 'New in 4.3' was still lingering in the 4.4 doc. Removed those as well.